src/cheri_insts.sail

```diff
@@ -596,6 +596,30 @@ function clause execute (CSetBoundsExact(cd, cs1, rs2)) = {
   RETIRE_SUCCESS
 }
 
+union clause ast = CSetBoundsRoundDown : (regidx, regidx, regidx)
+/*!
+ * Capability register *cd* is set to capability register *cs1* with its
+ * **base** field replaced with *cs1*.**address** and its **length** field
+ * replaced with the maximum exactly representable value no larger than
+ * the value in the integer register *rs2*.
+ *
+ * ## Notes
+ *  - The same caveat regarding the order of the bounds check applies as for
```

Comment:
I think the caveat doesn't actually apply because the bounds are always a subset of the requested bounds.

Reply:
How would you feel about...

```
 *  - As with [CSetBounds], the implementation here does the bounds check on the
 *    *requested* bounds.  This is less interesting here than there because the
 *    results of [setCapBoundsRoundDown] are always a subset of the requested
 *    range.
```